src/org/loklak/api/search/LocationWiseTimeService.java

```diff
@@ -21,15 +21,15 @@
 package org.loklak.api.search;
 
 import java.io.IOException;
-
 import javax.servlet.http.HttpServletResponse;
-
-import org.json.JSONArray;
-import org.json.JSONObject;
 import org.jsoup.Jsoup;
 import org.jsoup.nodes.Document;
 import org.jsoup.nodes.Element;
 import org.jsoup.select.Elements;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
```

Comment:
@kavithaenair @singhpratyush , there a rule to keep imports from project and from other projects separate, isn't it?
`org.json` belongs to loklak server project and `org.jsoup` is other project's import